Configure

```diff
@@ -1203,18 +1205,16 @@ unless ($disabled{engine}) {
             ($mi2) = $mi2 =~ /(\d+)/;
             my $ver = $ma*10000 + $mi1*100 + $mi2;
             if ($ver < $minver) {
-                $disabled{afalg} = "too-old-kernel";
+                $disabled{afalgeng} = "too-old-kernel";
             } else {
-                push @{$config{engdirs}}, "afalg";
+                push @{$config{engdirs}}, "afalgeng";
```

Comment:
This was going too far.  `$config{engdirs}` is the collection of `engines/` subdirs to go into (classic build scheme only, the unified scheme doesn't give a damn about this particular config).